docs/references/supported-signatures.mdx

```diff
@@ -0,0 +1,34 @@
+---
+keywords: [advanced, multi-chain, multichain, signatures, tecdsa, ecdsa, cryptography ]
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+
+# Supported signatures
+
+<MarkdownChipRow labels={["Reference"]} />
+
+## Currently supported signatures
+
+ICP currently supports the following cryptographic signature types:
+
+- [ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa): ECDSA is supported via the ICP's novel threshold ECDSA protocol. Threshold ECDSA uses private ECDSA keys that only exist as secret shares that are distributed across the nodes within the threshold ECDSA subnet. Signatures are generated by computing these secret shares without reconstructing the private key. [Learn more about t-ECDSA](/docs/current/developer-docs/smart-contracts/encryption/t-ecdsa)
+
+- Schnorr: Schnorr signatures are supported for processing certain types of assets, such as Ordinals and BRC20 assets, and enable web-based smart contract wallets such as [Oisy wallet](https://oisy.com/).
+
+- Boneh-Lynn-Shacham (BLS): ICP's chain-key cryptography is based on BLS signatures. While BLS signatures have distinct benefits and advantages, they cannot be used on other blockchains. To integrate with other chains, ICP uses threshold signatures like threshold ECDSA. 
+
+## Future additions
+
+The following signatures are planned to be supported in the future:
+
+- EdDSA
+
+- Multi-signature
```

Comment:
What's this?

Reply:
Mentioned in this FAQ page as something "not yet" supported: https://internetcomputer.org/docs/current/developer-docs/multi-chain/faq/signatures-faq#does-icp-support-eddsa-signatures

Reply:
I'd focus on currently supported "threshold signatures" and only mention Schnorr and EdDSA in the future addition section.

Furthermore, we should mention that BLS signatures are used for Canister Signatures/Certified variables: https://internetcomputer.org/docs/current/concepts/glossary/#canister-signature.
(and plan for a proper page about canister signatures in the future)